core/toolbox/toolbox.js

```diff
@@ -545,6 +543,7 @@ Blockly.Toolbox.prototype.position = function() {
   if (this.horizontalLayout_) {
     toolboxDiv.style.left = '0';
     toolboxDiv.style.height = 'auto';
+    // TODO: Double check that 100% instead of the svgSize.width is not a problem.
```

Comment:
These TODOs should be replaced with issue numbers (but I recognize that this is a feature branch, just keep a note to do so before this goes into develop).

Reply:
Yup, will do : )